pkg/controller.v1alpha3/trial/trial_controller_util.go

```diff
@@ -91,23 +91,27 @@ func (r *ReconcileTrial) UpdateTrialStatusCondition(instance *trialsv1alpha3.Tri
 	now := metav1.Now()
 	jobConditionType := (*jobCondition).Type
 	if jobConditionType == commonv1.JobSucceeded {
-		msg := "Trial has succeeded"
-		instance.MarkTrialStatusSucceeded(TrialSucceededReason, msg)
-		instance.Status.CompletionTime = &now
+		if isTrialObservationAvailable(instance) {
+			msg := "Trial has succeeded"
+			instance.MarkTrialStatusSucceeded(TrialSucceededReason, msg)
+			instance.Status.CompletionTime = &now
+
+			eventMsg := fmt.Sprintf("Job %s has succeeded", deployedJob.GetName())
+			r.recorder.Eventf(instance, corev1.EventTypeNormal, JobSucceededReason, eventMsg)
+		} else {
+			eventMsg := fmt.Sprintf("Metrics are not available for Job %s", deployedJob.GetName())
+			r.recorder.Eventf(instance, corev1.EventTypeWarning, MetricsUnavailableReason, eventMsg)
```

Comment:
Maybe we could add a condition to show the warning.

Reply:
I will add it